tests/serde.rs

```diff
@@ -11,6 +11,10 @@ macro_rules! test_roundtrip {
             assert_eq!(json_string, expected_json);
             let recover: $ty = serde_json::from_str(&json_string).unwrap();
             assert_eq!(&*tiny, &*recover);
+
+            let bin = bincode::serialize(&tiny).unwrap();
```

Comment:
Consider putting the bincode bytes themselves into the test file like you did with JSON.

Reply:
Done!